here-naksha-lib-psql/src/test/java/com/here/naksha/lib/psql/PsqlTests.java

```diff
@@ -142,6 +147,26 @@ static void beforeTest() {
     NakshaContext.currentContext().setAppId("naksha-lib-psql-unit-tests");
     nakshaContext = new NakshaContext().withAppId(TEST_APP_ID).withAuthor(TEST_AUTHOR);
     fg = new PsqlFeatureGenerator();
+    final String url;
+    if (existingUrl != null) {
+      url = existingUrl;
+    } else {
+      final DockerImageName image = DockerImageName.parse("greenoag/postgres-plv8-postgis:15.2-3.1.5-3.3")
+                      .asCompatibleSubstituteFor("postgres");
```

Comment:
I know the target of this PR is not yet our main branch but in the future we need to have single common docker image with configuration that is used for tests / local development.
Currently, with this one - there are two (see `PostgresContainer` in `naksha-app-service`). I think we can live with that schism for now but once plv8 will be required, we should align these two and have common container setup for all test that rely on running DB

Reply:
Makes sense, but in any case, we have this already on alw_jbon (see `Plv8TestContainer` in `here-naksha-lib-plv8`).

Reply:
I will add a separate ticket to unify these two, don't bother addressing this now